build/depends.py

```diff
@@ -111,6 +111,22 @@ def configure(self, build, conf):
         build.env.ParseConfig(
                 'pkg-config upower-glib --silence-errors --cflags --libs')
 
+class CtlraLibrary(Dependence):
+    """Ctlra library is used to access modern controller devices"""
+    def configure(self, build, conf):
+        if not build.platform_is_linux:
+            return
+        ctlra_env = str(os.environ["CTLRA_PATH"])
```

Comment:
Yes absolutely. I don't actually have experience writing pkg-config files, and while its meant to be easy, I didn't find an easy to follow tutorial / example with CMake how to do it properly. I'll try / keep digging though, and ultimately when Ctlra is upgraded then this can follow suit.